### DISCUSSION TITLE
[Curriculum] As a teacher logged in to terrastories, I can add stories to a given curriculum

### DISCUSSION THREAD

**Discussion by FeminismIsAwesome** [Terrastories/terrastories#733] [Temporarily Archived Tickets]
![image](https://user-images.githubusercontent.com/5641692/78455005-af603300-7650-11ea-992c-b1abd9024523.png)

For this issue:
When managing a specific Curriculum, I can search the stories by name, and select one of those stories to add it to a given Curriculum.

![image](https://user-images.githubusercontent.com/5641692/78455159-7ffdf600-7651-11ea-893d-5ffad2af8364.png)

==NOTE== skipping the display of the previously selected stories in a curriculum, that will be handled by a seperate issue.

 In the future, it would be great to add stories directly from the map view. But for now, we can begin with the architecture where we can search for stories by name.

Context from Rudo, high level:
Let’s say there are 500 stories in Terrastories for a community. This is the entire archive of this community’s oral histories. But today, a teacher wants to focus on just a few of them that are thematically related, and play them a sequential order for her classroom.

The teacher can log in to Administrate to create a new Curriculum, select those three stories, and then activate the Curriculum. Then, when the teacher goes back to the map view of Terrastories, only those three stories (and associated places) will show up.

I will illustrate how this should work in Administrate using wireframes, below.
In Administrate, there will be a Curriculum menu that admin users can access. This is what it looks like.

**Comment by FeminismIsAwesome** [comment]
@rudokemper can you see if recent commit merges by sarah's work do this well enough to accept this issue or if we need more work for a first pass?

**Comment by rudokemper** [comment]
@FeminismIsAwesome @slaloggia I was able to add stories to the curriculum! 

I only wonder about the mechanism that we use to do so, and the UI. Right now it's an autotag list (there probably is a more technical term), but it would be nice for them to show them in a table format with more information about the stories beside the name, as per the wireframes above.

But I might be getting ahead of myself. If the purpose of this ticket is just to be able to add stories to the curriculum and not worry about the UI just yet, this is great 👍
